modules/mbedtls/CMakeLists.txt

```diff
@@ -51,6 +50,9 @@ else()
   # Lib mbedtls_external depends on libgcc (I assume?) so to allow
   # mbedtls_external to link with gcc we need to ensure it is placed
   # after mbedtls_external on the linkers command line.
+else()
+  # You are on your own with regards to include folders and library for
+  # mbed TLS.
```

Comment:
The reason and logic is sane, but this should be rephrased before sending upstream, imho.

Reply:
Will fix this @tejlmand

Reply:
fixed.